mula/tests/integration/test_api.py

```diff
@@ -513,7 +514,16 @@ def test_pop_queue_filters(self):
         # Should not return any items
         response = self.client.post(
             f"/queues/{self.scheduler.scheduler_id}/pop",
-            json={"filters": [{"column": "data", "field": "id", "operator": "eq", "value": "123"}]},
+            json={
```

Comment:
linter back on `black`? ;)

Reply:
Added a `,` to the end it automatically formats it to superior column width, and precommit even agrees. *old-man-shakes-fist* You whippersnappers with your wide monitors!